criteria/understandable-english-first.md

```diff
@@ -6,10 +6,11 @@ order: 10
 
 ## Requirements
 
-* All code and documentation MUST be in English.
+* All codebase documentation MUST be in English.
+* All code MUST be in English, except where required to execute as policy.
```

Comment:
I think we could clarify this even further. Perhaps "All code MUST be in English, except where policy is required to execute as code." is a better wording?

Reply:
Indeed this should be sharper; we want a clearer way to express that where policy is directly interpreted as code, or when code is word-for-word from policy, then it will naturally be in the language of that policy.

Reply:
perhaps "except where policy is interpreted directly as code" ?

Reply:
I think that may be the most of what we are actually trying to capture. "All code MUST be in English, except when code is word-for-word from policy." That would be a requirement in more plain English,

Reply:
Yes, your "interpreted" suggestion is also good!